protostar/cheatable_starknet/cheatables/cheatable_cached_state.py

```diff
@@ -77,6 +80,14 @@ def remove_pranked_address(self, target_address: Address):
     def get_block_info(self, contract_address: int) -> BlockInfo:
         return BlockInfoController(self).get_for_contract(Address(contract_address))
 
+    async def set_contract_class(
+        self, class_hash: int, contract_class: CompiledClassBase
+    ):
+        self.contract_classes[class_hash] = contract_class
+
+    async def get_contract_class(self, class_hash: int) -> CompiledClassBase:
+        return await self.get_compiled_class(class_hash)
+
```

Comment:
Wrappers to make `state` related code backwards-compatible, since `CachedState` changed implementation:
`starkware.starknet.business_logic.state.state.CachedState`